config/routes.rb

```diff
@@ -6,6 +6,7 @@
   post 'comment/create/token/:id.:format', to: 'comment#create_by_token'
 
   get 'searches/test' => 'searches#test'
+  post '/reset-title' => 'notes#request_title_change'
```

Comment:
Does this actually change the title, or does it request a title change? I think it actually changes the title. Maybe we could do like `/node/update/title` or something more descriptive? Same for the method name `request_title_change` -- could that be `update_title` to be more accurate, or do i misunderstand?

Reply:
This actually changes the title. Yes, that's a cool idea!! I'll change the route name and method name to the above.